combat-trainer.lic

```diff
@@ -1910,14 +1919,38 @@ class GameState
     @use_weak_attacks = false
   end
 
+  def use_analyze_flame?
+    @use_analyze_flame
+  end
+
+  def stop_analyze_flame
+    @use_analyze_flame = false
+  end
+
   def melee_attack_verb
```

Comment:
Actually, I think the Brawling and Offhand checks work for this without doing anything. It will just simply bypass the case until you are no longer brawling or using an offhand.